### PR TITLE
doc: Add troubleshooting for UDP packet forwarder and devices

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding a troubleshooting section for UDP packet forwarder, with start on the failing downlinks. Also adding a short point on troubleshooting devices.
Ref: #208

#### Notes for Reviewers
@johanstokking sanity check please. I addressed only what was instructed at the moment, but with some guidance I can research and add more info. Also, I'm not sure how can a person change the device's link check behavior?

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### No downlinks are reaching my UDP gateway. Why?
+
+With UDP gateways, if uplink messages are reaching {{% tts %}}, that does not mean the downlink messages are going to be received correctly by the gateway. 
+
+The {{% udp-pf %}} sends a `PULL_DATA` request every ~5 seconds to pull the data scheduled for a downlink transmission on {{% tts %}}.
+
+If downlink messages are reaching your gateway, the {{% udp-pf %}} should be sending `PULL_ACK` messages with a reasonable latency, usually way lower than 500 ms. If this latency is high, pulling the downlink data will fail. One of the drawbacks of using the {{% udp-pf %}} is that it does not have an implemented mechanism to detect if pulling the downlink data fails, so your data might end up lost. 
+
+Common reasons for having latency issues between the gateway and {{% tts %}} Network Server are using the router that is too far from your location, and using cellular or satellite backhauls for gateways.
```
Comment (benolayinka) [inline doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md:18]:
```suggestion
Common reasons for having latency issues between the gateway and {{% tts %}} Network Server are that the gateway is geographically far from the network server, and using cellular or satellite backhauls for gateways.
```

File: doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Try solving this issue by: 
+
+- Checking your internet connection
+- Using the router closest to the location of your gateway
```
Comment (benolayinka) [inline doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md:23]:
```suggestion
- Using a network server closest to the location of your gateway
```

**Review by benolayinka — APPROVED** [review]

File: doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md (hunk 1 above)
Comment (johanstokking) [reply doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md:18]: Inconsistent casing

File: doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md (hunk 2 above)
Comment (johanstokking) [reply doc/content/gateways/semtech-udp-packet-forwarder/troubleshooting.md:23]: Network Server, but maybe `{{% tts %}} cluster` is better?

**Comment by nejraselimovic** [comment]
@johanstokking anything else to add?

**Comment by benolayinka** [comment]
> @johanstokking anything else to add?

you're good. Squash fixups and merge.

**Review by johanstokking — APPROVED** [review]
Great, doesn't have to wait for me indeed.

**Pull request merged**